packages/perseus/src/interactive2/wrapped-ellipse.ts

```diff
@@ -13,82 +14,90 @@ const DEFAULT_OPTIONS = {
     disableMouseEventsOnWrapper: false,
 } as const;
 
-const WrappedEllipse = function (
-    graphie: any,
-    center: Coord,
-    radii: [number, number],
-    options: any,
-) {
-    options = _.extend({}, DEFAULT_OPTIONS, options);
+class WrappedEllipse extends WrappedDefaults {
+    graphie: Graphie;
+    initialPoint: Coord;
+    wrapper: HTMLDivElement;
+    visibleShape: {
```

Comment:
The base class already has `graphie`, `wrapper`, and `visibleShape`.  This feels risky/unnecessary for the subclasses to also define them.